### PR TITLE
add nfs-utils

### PR DESCRIPTION
**Pull request by rongrongqiu**
1.Upgrade automake to 1.15
2.Add package libtirpc-0.3.2
3.Add package libnfsidmap-0.25
4.Add package nfs-utils-1.3.2

### CODE REVIEW

**Comment by rongrongqiu** [comment]
Please add commit "modify sources_list.sha1 for nfs-utils". Thanks!

File: SPECS/automake/automake.spec [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-Release:	2%{?dist}
+Version:	1.15
+Release:	1%{?dist}
 License:	GPLv2+
 URL:		http://www.gnu.org/software/automake/
 Group:		System Environment/Base
 Vendor:		VMware, Inc.
 Distribution: 	Photon
-Source0:	http://ftp.gnu.org/gnu/automake/%{name}-%{version}.tar.xz
+Source0:	%{name}-%{version}.tar.gz
```
Comment (toliaqat) [inline SPECS/automake/automake.spec]: Please specify the complete URL from which we got this source tar file.

File: SPECS/libnfsidmap/libnfsidmap.spec [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Summary:	libnfsidmap
+Name:		libnfsidmap
+Version:	0.25
+Release:	1
+License:	GPLv3+
+URL:		http://www.linuxfromscratch.org/
+Group:		Applications/libnfsidmap
+Source0:	%{name}-%{version}.tar.gz
```
Comment (toliaqat) [inline SPECS/libnfsidmap/libnfsidmap.spec]: Ditto

File: SPECS/libtirpc/libtirpc.spec [hunk 3]
```diff
@@ -0,0 +1,42 @@
+Summary:	libtirpc
+Name:		libtirpc
+Version:	0.3.2
+Release:	1
+License:	GPLv3+
+URL:		http://www.linuxfromscratch.org/
+Group:		Applications/Libtirpc
+Source0:	%{name}-%{version}.tar.bz2
+Patch0:          libtirpc-0.3.2-api_fixes-1.patch
```
Comment (toliaqat) [inline SPECS/libtirpc/libtirpc.spec]: Dito

File: docs/nfs-utils.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Photon NFS utils
+
+To mount network file in Photon
```
Comment (toliaqat) [inline docs/nfs-utils.md]: Mention that its installed by default in FULL installation of Photon and Minimial use tdnf (mention complete command to install nfs on minimal) command to install it.

File: docs/nfs-utils.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+## Check a remote nfs server
+showmount  -e nfs-servername or ip
+
+for example : showmount  -e discus-filer.eng.vmware.com
+
+              showmount -e 10.118.100.122
+
+## Mount the nfs
+
+mount -t nfs nfs-servername:/export /mnt
```
Comment (toliaqat) [inline docs/nfs-utils.md]: Use ''' ''' markdown tags to make the code look good.

File: support/pullsources/sources_list.sha1 [hunk 6]
```diff
@@ -219,6 +223,7 @@ tcl8.6.1-src.tar.gz - 5c83d44152cc0496cc0847a2495f659502a30e40
 tcpdump-4.7.3.tar.gz - 4f085cef7cd4aedc9e402021ec11e3a8b23a6926
 tcsh-6.18.01.tar.gz - eee2035645737197ff8059c84933a75d23cd76f9
 tdnf-1.0.1.tar.gz - bff1ffdce4f7c319c06da3a76f8be70f76be1246
+tdnf-1.0.tar.gz - f2e10a64fd564082c64eb4e4b585d7192cf47ff5
```
Comment (toliaqat) [inline support/pullsources/sources_list.sha1]: Please make sure you are not adding any extra file which you did not intended to be added. TDNF was added and removed by Priyesh so make sure you do not upload it again.

File: SPECS/libnfsidmap/libnfsidmap.spec [hunk 7]
```diff
@@ -0,0 +1,32 @@
+Summary:	libnfsidmap
+Name:		libnfsidmap
+Version:	0.25
+Release:	1
```
Comment (toliaqat) [inline SPECS/libnfsidmap/libnfsidmap.spec]: Please add 1%{?dist} in release.,

File: SPECS/libtirpc/libtirpc.spec [hunk 8]
```diff
@@ -0,0 +1,42 @@
+Summary:	libtirpc
+Name:		libtirpc
+Version:	0.3.2
+Release:	1
```
Comment (toliaqat) [inline SPECS/libtirpc/libtirpc.spec]: 1%{?dist}

File: SPECS/nfs-utils/nfs-utils.spec [hunk 9]
```diff
@@ -0,0 +1,41 @@
+Summary:	Nfs service
+Name:		nfs-utils
+Version:	1.3.2
+Release:	1
```
Comment (toliaqat) [inline SPECS/nfs-utils/nfs-utils.spec]: 1%{?dist}

**Pull request closed without merging**